<launch>

  <arg name="hand_control_topic" default="/allegroHand/joint_cmd" />
  <arg name="synergy_bag_file" />
  <arg name="num_synergies" default="5" />
  <arg name="synergy_gui" default="false" />
  <arg name="rqt_slider_config"
       default="$(find grasp_synergy)/launch/rqt_ez_pub-config.yaml" />

  <node name="grasp_synergy_node"
        pkg="grasp_synergy"
        type="synergy_node.py"
        output="screen"
        args="--hand_control_topic $(arg hand_control_topic)
              --bag_filename $(arg synergy_bag_file)
              --num_synergies $(arg num_synergies)" />

  <node name="synergy_gui"
        pkg="rqt_ez_publisher"
        type="rqt_ez_publisher"
        args="--slider-file $(arg rqt_slider_config)"
        if="$(arg synergy_gui)"/>
</launch>
